chat-widget/src/components/confirmationpanestateful/ConfirmationPaneStateful.tsx

```diff
@@ -83,7 +83,7 @@ export const ConfirmationPaneStateful = (props: IConfirmationPaneStatefulParams)
 
     return (
         <>
-            {!controlProps?.disableDimLayer && <DimLayer brightness={controlProps?.brightnessValueOnDim ?? "0.2"} />}
```

Comment:
Why are we removing this controlProp?

Reply:
if disableDimLayer is true, and we open confirmation pane or email transcript pane, then other components in the background are still clickable